layouts/default.vue

```diff
@@ -19,6 +19,7 @@
               style="float: right"
             >
               <a
+                class="avatar-menu"
```

Comment:
@appinteractive do we use some sort of fancy CSS naming like BEM?

Reply:
Yeah kinda, as you can see we prefix the styles with the component name. But not actually in the ugly BEM style.